tests/test_utils.py

```diff
@@ -11,74 +9,93 @@
 from dbignite.schemas import *
 
 REPO = os.environ.get("REPO", "dbignite")
-BRANCH = re.sub(r"\W+", "", os.environ.get("BRANCH", 'local_test'))
-
 TEST_BUNDLE_PATH = "./sampledata/"
-TEST_DATABASE = f"test_{REPO}_{BRANCH}"
-
-
-@pytest.fixture
-def get_entries_df(spark_session):
-    entries_df = FhirBundlesToCdm(spark_session).loadEntries(TEST_BUNDLE_PATH)
-    return(entries_df)
-
-@pytest.fixture
-def fhir_model():
-    fhir_model=FhirBundles(TEST_BUNDLE_PATH)
-    return(fhir_model)
-
-@pytest.fixture
-def cdm_model():
-    cdm_model=OmopCdm(TEST_DATABASE)
-    return(cdm_model)
-
-class TestUtils:
-
-    def test_entries_to_person(self,get_entries_df) -> None:
-        person_df = entries_to_person(get_entries_df)
+TEST_DATABASE = f"test_{REPO}"
+
+
+class TestUtils(unittest.TestCase):
```

Comment:
I think the pattern is to use @pytest.fixture for tests

Reply:
Because our code is class based I think we should use Unittest to align for class based test execution. pytest package also runs Unittest classes as well. 

e.g. Solution Accelerator Waterbear also uses Unittest. I'm not bound to using Unittest, so I'm happy to change this back. However, I think we should remove spark testing packages as I don't see what functional value they are adding and one doesn't appear to be maintained any longer.